benchmark.sh

```diff
@@ -30,8 +34,4 @@ printf "\nmemsweep:\n">> results.txt
 sh memsweep.sh | grep "seconds" >> results.txt
 
 printf "\nHardware:\n" >> results.txt
-sudo lshw -short >> results.txt
-
-
-rm linpack
```

Comment:
the compiled versions might interfere with rsync and are therefore deleted beforehand